Jenkinsfile

```diff
@@ -370,9 +415,16 @@ pipeline {
                                 }
                             }
                         }
+                        job_status_update()
                     } // steps
-                } // stage ('Commit Pragma tests') {
+                } // stage ('Commit Pragma tests') */
```

Comment:
Where does the comment that this line is ending start?

Reply:
Missed edit, will remove on next push.